src/main/java/com/google/research/bleth/simulator/AbstractSimulation.java

```diff
@@ -129,7 +128,8 @@ void writeSimulationStats() {
         for (Beacon beacon : beacons) {
             String beaconId = String.valueOf(beacon.getId());
             beaconsObservedSum.putIfAbsent(beaconId, 0D);
-            beaconsObservedSum.put(beaconId, beaconsObservedSum.get(beaconId) / (currentRound - 1));
+            beaconsObservedSum.put(beaconId, beaconsObservedSum.get(beaconId)
```

Comment:
Consider splitting the computation to a separate variable (and then a properly-chosen variable name can obviate the need for a comment).